packages/webapp/pages/notifications.tsx

```diff
@@ -1,20 +1,58 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
 import { NextSeo } from 'next-seo';
+import { useInfiniteQuery, InfiniteData, useMutation } from 'react-query';
+import {
+  NotificationsData,
+  NotificationType,
+  NOTIFICATIONS_QUERY,
+  READ_NOTIFICATIONS_MUTATION,
+} from '@dailydotdev/shared/src/graphql/notifications';
 import {
   pageContainerClassNames,
   pageBorders,
 } from '@dailydotdev/shared/src/components/utilities';
+import request from 'graphql-request';
+import { apiUrl } from '@dailydotdev/shared/src/lib/config';
 import NotificationItem from '@dailydotdev/shared/src/components/notifications/NotificationItem';
 import EnableNotification from '@dailydotdev/shared/src/components/notifications/EnableNotification';
 import { cloudinary } from '@dailydotdev/shared/src/lib/image';
-import { NotificationType } from '@dailydotdev/shared/src/graphql/notifications';
-import { getLayout as getFooterNavBarLayout } from '../components/layouts/FooterNavBarLayout';
-import { getLayout } from '../components/layouts/MainLayout';
+import {
+  getLayout,
+  getLayout as getFooterNavBarLayout,
+} from '../components/layouts/FooterNavBarLayout';
 import ProtectedPage from '../components/ProtectedPage';
 
+const checkHasUnread = (data: InfiniteData<NotificationsData>) =>
```

Comment:
```suggestion
const hasUnread = (data: InfiniteData<NotificationsData>) =>
```

Reply:
Or `hasUnreadNotifications`

Reply:
I believe we use `has` as the first word on a variable, but to let it be known as a function we put some kind of prefix for it. Though I am happy to change this as well if it is actually not a usual thing in our codebase.